lib/trino-filesystem/pom.xml

```diff
@@ -38,7 +38,7 @@
         </dependency>
 
         <dependency>
-            <groupId>io.opentelemetry</groupId>
+            <groupId>io.opentelemetry.semconv</groupId>
```

Comment:
This commit can be dropped.

Reply:
Yeah, I'm dropping it prior to the merge.